td-shim/src/event_log.rs

```diff
@@ -136,32 +136,34 @@ impl ctx::TryIntoCtx<Endian> for &TpmlDigestValues {
 
 #[repr(C)]
 #[derive(Default, Debug, Pread, Pwrite)]
-pub struct TcgPcrEvent2Header {
-    pub pcr_index: u32,
+pub struct CcEventHeader {
+    pub mr_index: u32,
     pub event_type: u32,
     pub digest: TpmlDigestValues,
     pub event_size: u32,
 }
 
 #[repr(C, packed)]
 #[derive(Default, AsBytes, FromBytes)]
-pub struct Tdel {
+pub struct Ccel {
     pub header: GenericSdtHeader,
-    pub reserved: u32,
+    pub cc_type: u8,
+    pub cc_subtype: u8,
+    pub reserved: u16,
     pub laml: u64,
     pub lasa: u64,
 }
 
-impl Tdel {
-    pub fn new(laml: u64, lasa: u64) -> Tdel {
-        let mut tdel = Tdel {
-            header: GenericSdtHeader::new(b"TDEL", size_of::<Tdel>() as u32, 1),
+impl Ccel {
+    pub fn new(laml: u64, lasa: u64) -> Ccel {
+        let mut ccel = Ccel {
+            header: GenericSdtHeader::new(b"CCEL", size_of::<Ccel>() as u32, 1),
             laml,
             lasa,
             ..Default::default()
```

Comment:
How you initialize the cc_type?

Reply:
Missed this part, add two arguments to specify cc_type and cc_subtype for the new method